src/main/java/com/kodilla/ecommercee/domain/OrderEntity.java

```diff
@@ -1,14 +1,15 @@
 package com.kodilla.ecommercee.domain;
 
+import com.kodilla.ecommercee.User;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
+import javax.persistence.*;
```

Comment:
Importy z gwiazdką z tego co kojarzę Mateusz odradzał, można rozbić na pojedyncze importy. Jak myślisz?

Reply:
odradzałem, to fakt :-) wiem, że można stracić na tym trochę nerwów - szczególnie w początkowej fazie przygody z programowaniem.

Reply:
Faktycznie jest to denerwujące w przypadku dodawania ostatniego importu, który automatycznie zmienia wszystkie inne persistence w jeden z gwiazdką. Ale poradziłem jakoś sobie.